packages/terra-functional-testing/lib/services/wdio-asset-service.js

```diff
@@ -0,0 +1,45 @@
+const ExpressServer = require('../express-server/express-server');
+const WebpackServer = require('../webpack-server/webpack-server');
+const Logger = require('../logger/logger');
+
+const logger = new Logger({ prefix: 'wdio-asset-service' });
```

Comment:
Should this be name spaced with terra?

Reply:
Do you have concerns about issues that might arise without a prefix? My thoughts on this is they are going to be scoped by:

```
@cerner/terra-functional-testing/path/wdio-asset-service
```

Reply:
I'm confused. What is the logged output here?

`[@cerner/terra-functional-testing/path/wdio-asset-service] log message...`

Reply:
The log output is going to be:

```
[terra-functional-testing:wdio-asset-service] Log message....
```

The import, if needed, would be:

```js
const AssetService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-service');
```